application/tests/spreadsheet_test.py

```diff
@@ -4,7 +4,7 @@
 from application import create_app, sqla  # type: ignore
 from application.database import db
 from application.defs import cre_defs as defs
-from application.utils.spreadsheet import prepare_spreadsheet
+from application.utils.spreadsheet import prepare_spreadsheet, generate_staggered_csv
```

Comment:
shouldn't we use more functional naming? so instead of staggered_csv say mappingtemplatefile?